changelog.md

```diff
@@ -0,0 +1,28 @@
+## 1.1.0 02/2024
+
+### Features
+* Poll in browser
+
+### Fixes
+* Separate the css and js from the view file
+* Fix the sealing violation issue occurred on jetty
+* Compatibility with the IdP 5.0.0 and higher
```

Comment:
If it is not compatible with IdP 4.x.x anymore, we should state that here just to be clear.